evmcc/evmcc.go

```diff
@@ -152,6 +157,9 @@ func (evmcc *EvmChaincode) Invoke(stub shim.ChaincodeStubInterface) pb.Response
 
 		// Passing the function hash of the method that has triggered the event
 		// The function hash is the first 8 bytes of the Input argument
+		// Since the argument is already hex encoded, we can directly pass the bytes, so
```

Comment:
A bit more about the why here would help me.

"The argument is a hex-encoded evm function hash, so we can directly pass the bytes"